### ISSUE TITLE
Oculus Quest controller event handling

### ISSUE THREAD

**Issue by dirkk0** [aframevr/aframe#4267]
**Description:**
The documentation provides an example on how to handle controller events:
https://aframe.io/docs/0.9.0/introduction/interactions-and-controllers.html#listening-for-button-and-axis-events

This doesn't work (for me?) on Oculus Quest.
The good news: event handling *does* work by directly addressing the controllers and attaching event listeners, as provided in the code pen below.

So I am not sure if I 
a) simply did something wrong or
b) the documentation isn't correct or
c) the Quest support is lacking at this point.

Usually I go with a) :).

- A-Frame Version:
0.9.2
- Platform / Device:
Oculus Quest
- Reproducible Code Snippet or URL:
https://codepen.io/dirkk0/pen/MWgGyJY
In this example, buttons A, B and Y are addressed directly, and X is handled like the example in the docs.  X doesn't work, the others do.

**Comment by chetu3319** [comment]
Hi @dirkk0 

TLDR : https://glitch.com/~ab-b

Few things  in your code : 
 - Wait for the dom elements to load before accessing them via `document.getElementById`. When your script will run to fetch elements `ctlL`, `cltR` and `txt` it will get null elements. And so the events listeners won't bind. For the fix, I added `windows.onload`.
- In the `x-button-listener` component use `this.el.eventListener`.  Just `el` represents nothing. 

Thanks,
Chait
